### PR TITLE
EJ2-70478 : How to navigate to the next and previous signature fields

### PR DESCRIPTION
**Pull request by SaravanaPriya31**
### Description

How to navigate to the next and previous signature fields

### CODE REVIEW

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/navigation.md [hunk 1]
```diff
@@ -87,6 +87,22 @@ You can change the open state of the hyperlink in the PDF Viewer by using the fo
 
 ![Alt text](./images/toc.png)
 
+## Keyboard navigation with Tab and Shift+Tab keys
+
+The Syncfusion PDF Viewer component supports keyboard navigation for the form fields in a PDF document using the `Tab` and `Shift+Tab` keys. This allows users to easily navigate through the form fields in a PDF document, making it easier for them to fill out forms.
```
Comment (florenceliliana) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/navigation.md:92]: The Syncfusion PDF Viewer component supports keyboard navigation for the form fields in a PDF document using the `Tab` and `Shift+Tab` keys. This allows users to easily navigate through the form fields in a PDF document, making it easier to fill out forms.

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+control: How to navigate to the next and previous signature fields
+publishingplatform: ##Platform_Name##
+documentation: ug
+---
+
+# How to navigate to the next and previous signature fields
+
+To navigate to the next and previous signature fields programmatically in Syncfusion PDF Viewer.
+
+The `previousField()` method is used to move the focus to the previous signature field, and the `nextField()` method is used to move the focus to the next signature field.
```
Comment (florenceliliana) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md:15]: The `previousField()` method moves the focus to the previous signature field, and the `nextField()` method moves the focus to the next signature field.

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+description: Learn how to navigate to the next and previous signature fields in Syncfusion ##Platform_Name## Pdfviewer component of Syncfusion Essential JS 2 and more.
+platform: ej2-asp-core-mvc
+control: How to navigate to the next and previous signature fields
+publishingplatform: ##Platform_Name##
+documentation: ug
+---
+
+# How to navigate to the next and previous signature fields
+
+To navigate to the next and previous signature fields programmatically in Syncfusion PDF Viewer.
```
Comment (florenceliliana) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md:13]: Navigate to the next and previous signature fields programmatically in the Syncfusion PDF Viewer.

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: How to navigate to the next and previous signature fieldsin ##Platform_Name## Pdfviewer Component
+description: Learn how to navigate to the next and previous signature fields in Syncfusion ##Platform_Name## Pdfviewer component of Syncfusion Essential JS 2 and more.
+platform: ej2-asp-core-mvc
+control: How to navigate to the next and previous signature fields
```
Comment (florenceliliana) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md:6]: control: How to navigate to the next and previous signature fields.

**Review by florenceliliana — CHANGES_REQUESTED** [review]

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/navigation.md (hunk 1 above)
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/navigation.md:92]: The Syncfusion PDF Viewer component supports keyboard navigation for the form fields in a PDF document using the `Tab` and `Shift+Tab` keys. This allows users to easily navigate through the form fields in a PDF document, making it easier to fill out forms.

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
+---
+layout: post
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md:2]: Layout: Post

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+title: How to navigate to the next and previous signature fields ##Platform_Name## Pdfviewer Component
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md:3]: Title: How to navigate to the next and previous signature fields ##Platform_Name## Pdfviewer Component

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+description: Learn how to navigate to the next and previous signature fields in Syncfusion ##Platform_Name## Pdfviewer component of Syncfusion Essential JS 2 and more.
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md:4]: Description: Learn how to navigate to the next and previous signature fields in Syncfusion ##Platform_Name## Pdfviewer component of Syncfusion Essential JS 2 and more.

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+platform: ej2-asp-core-mvc
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md:5]: Platform: ej2-asp-core-mvc

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+control: How to navigate to the next and previous signature fields
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md:6]: Control: How to navigate to the next and previous signature fields

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+publishingplatform: ##Platform_Name##
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md:7]: Publishingplatform: ##Platform_Name##

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+documentation: ug
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md:8]: Documentation: UG

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+---
+
+# How to navigate to the next and previous signature fields
+
+Navigate to the next and previous signature fields programmatically in the Syncfusion PDF Viewer.
+
+The `previousField()` method moves the focus to the previous signature field, and the `nextField()` method moves the focus to the next signature field.
+
+To use these functions, you need to ensure that the Syncfusion PDF Viewer is properly initialized on the page, and that the formFieldsModule module is loaded.
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md:17]: To use these functions, ensure the Syncfusion PDF Viewer is properly initialized on the page and the formFieldsModule module is loaded.

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+
+Here's an example of how to use the `previousField()` and `nextField()` method:
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md:19]: The following is an example of using the `previousField()` and `nextField()` methods.

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+    }
+    function nextField() {
+        var viewer = document.getElementById('pdfviewer').ej2_instances[0];
+        viewer.formFieldsModule.nextField();
+    }
+</script> 
+
+```
+
+The `previousField()` and `nextField()` methods are called on the formFieldsModule property of the viewer object when the corresponding buttons are clicked. When the functions are called, they use the `viewer.formFieldsModule` object to navigate between the form fields in the PDF document.
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/navigate-next-previous-signaturefields.md:39]: The `previousField()` and `nextField()` methods are called on the formFieldsModule property of the viewer object when the corresponding buttons are clicked. When the functions are called, they use the `viewer.formFieldsModule` object to navigate between the form fields in a PDF document.

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/navigation.md [hunk 15]
```diff
@@ -112,6 +112,22 @@ You can change the open state of the hyperlink in the PDF Viewer by using the fo
 
 ![Alt text](./images/toc.png)
 
+## Keyboard navigation with Tab and Shift+Tab keys
+
+The Syncfusion PDF Viewer component supports keyboard navigation for the form fields in a PDF document using the `Tab` and `Shift+Tab` keys. This allows users to easily navigate through the form fields in a PDF document, making it easier for them to fill out forms.
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/navigation.md:117]: The Syncfusion PDF Viewer component supports keyboard navigation for the form fields in a PDF document using the `Tab` and `Shift+Tab` keys. This allows users to easily navigate through the form fields in a PDF document, making it easier to fill out forms.

**Pull request closed without merging**